presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -93,6 +111,15 @@ public void testDynamicFilterSummaryCompletion()
         assertEquals(summary.get(), multipleValues(INTEGER, ImmutableList.of(1L, 2L, 3L)));
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 3);
 
+        stats = dynamicFilterService.getDynamicFilteringStats(queryId, session);
+        assertEquals(stats.getNumDynamicFiltersCompleted(), 1);
+        assertEquals(stats.getNumRepartitionedDynamicFilters(), 1);
```

Comment:
please also assert re-partitioned and replicated DF count after first `getDynamicFilteringStats` call